codegen/src/main/kotlin/software/amazon/smithy/rust/codegen/smithy/ServerRustSettings.kt

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package software.amazon.smithy.rust.codegen.smithy
+
+import software.amazon.smithy.model.Model
+import software.amazon.smithy.model.node.ObjectNode
+import software.amazon.smithy.model.shapes.ShapeId
+import java.util.Optional
+
+/**
+ * [ServerRustSettings] and [ServerCodegenConfig] classes.
+ *
+ * These classes are entirely analogous to [ClientRustSettings] and [ClientCodegenConfig]. Refer to the documentation
+ * for those.
+ *
+ * These classes have to live in the `codegen` subproject because they are referenced in [ServerCodegenContext],
+ * which is used in common generators to both client and server.
+ */
+
+/**
+ * Settings used by [RustCodegenServerPlugin].
+ */
+data class ServerRustSettings(
```

Comment:
No. This is called out a couple of lines above:

```
 *
 * These classes have to live in the `codegen` subproject because they are referenced in [ServerCodegenContext],
 * which is used in common generators to both client and server.
 */
```